content/en/serverless/aws_lambda/installation/dotnet.md

```diff
@@ -313,6 +313,15 @@ To use the optimized extension, disable Application Security Management (ASM), C
 
 Enabling any of these features cause the extension to default back to the fully compatible older version of the extension. You can also force your extension to use the older version by setting `DD_EXTENSION_VERSION` to `compatibility`. Datadog encourages you to report any feedback or bugs by adding an [issue on GitHub][8] and tagging your issue with `version/next`.
 
+## Adding custom spans
+
+If you are using both the Datadog tracer and the [Datadog Lambda tracing layer for .NET][9], you must ensure the code uses the same tracer instance for all instrumentation.
```

Comment:
We don't even document on how to instrument AWS Lambda without using the layer? Maybe we should explore figuring out how can customers instrument their function without needing to use the layer just like we do in other runtimes?